site-footer.md

```diff
@@ -0,0 +1,20 @@
+### Help & Support
+
+ Project Coordinators
+ 
+ [Beatriz Serrano-Solano](#)
```

Comment:
What will link it?

Reply:
I was still deciding here, that's why I asked in the matrix channel with the two options I had in mind:
(1) linking it to the github profiles of the coordinators, OR
(2) linking it to the matrix chat of the coordinators

We figured that (2) was a better idea since the intent is to offer support to the members.

So, I'll add the links now